### PR TITLE
fix typos in cpp documentation

### PR DESCRIPTION
**Pull request by garanews**
fix typos in cpp documentation

### CODE REVIEW

File: content/cpp/concepts/switch/switch.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 Title: 'Switch'
-Description: 'A switch statement provides a means of checking an expression against various case statements. If there is a match, the code within starts to execute. The break keyword can be used to terminate a case. Theres also an optional default statement marking code that executes if none of the case statements are true. A switch statement looks like: cpp switch (grade) { case 9: std::cout << "Freshman\n";'
+Description: 'A switch statement provides a means of checking an expression against various case statements. If there is a match, the code within starts to execute. The break keyword can be used to terminate a case. There is also an optional default statement marking code that executes if none of the case statements are true. A switch statement looks like: cpp switch (grade) { case 9: std::cout << "Freshman\n";'
```
Comment (SSwiniarski) [inline content/cpp/concepts/switch/switch.md:3]:
```suggestion
Description: 'A switch statement provides a means of checking an expression against various case statements.'
```

**Review by SSwiniarski — COMMENTED** [review]
@garanews, I did a review and left one comment.

**Comment by yangc95** [comment]
@garanews I took this PR as an opportunity to fix any additional issues with the C++ entries.

The main fix here is structuring the code blocks accordingly to the C++ standards.

@SSwiniarski, if you could take another look at this PR and look over the code blocks to make sure everything is accurate, that would be amazing (it's quite a bit to look over but I'll do a double-scan to make sure everything is consistent). Please let me know if anything is off. 🙏

**Review by SSwiniarski — APPROVED** [review]
@yangc95 Looks ok to me.

**Review by yangc95 — APPROVED** [review]

**Comment by yangc95** [comment]
@garanews thanks for these corrections! The edits should be visible across all [C++](https://www.codecademy.com/resources/docs/cpp) entries.

cc: @Dusch4593 @SSwiniarski

**Comment by garanews** [comment]
@yangc95  you're welcome!
Meanwhile, time ago, speaking with @Dusch4593 about adding new parts, he suggested me to open a discussion, I did it: there is someone of you had a time to look at it?
Thanks in advance!

**Pull request merged**